src/components/BookList.js

```diff
@@ -1,28 +1,11 @@
 import Book from './Book'
 import styled from 'styled-components/macro'
-import { useState } from 'react'
-
-function BookList({ books }) {
-  const [filteredBooks, setFilteredBooks] = useState(books)
-  const [readingStatus, setReadingStatus] = useState('')
-
-  function handleActiveReadingStatus(status, books) {
-    setReadingStatus(status)
-    if (status === 'true') {
-      const readBooks = books.filter(book => book.finished === true)
-      setFilteredBooks(readBooks)
-    } else if (status === 'false') {
-      const currentlyReadBooks = books.filter(book => book.finished === false)
-      setFilteredBooks(currentlyReadBooks)
-    } else if (status === '') {
-      setFilteredBooks(books)
-    }
-  }
 
+function BookList({ books, readingStatus }) {
   function filterTitle(status) {
-    if (status === 'true') {
+    if (status === 'finishedBooks') {
       return 'Your library of finished books:'
-    } else if (status === 'false') {
+    } else if (status === 'currentlyReading') {
       return 'Your library of books currently read:'
     } else {
       return ''
```

Comment:
Same here with the `else if` 🙂. You can change it to only `if`s for better readability.